examples/contrastive-image-text/README.md

```diff
@@ -108,7 +108,8 @@ python run_clip.py \
     --use_lazy_mode \
     --use_hpu_graphs \
     --gaudi_config_name Habana/clip \
-    --throughput_warmup_steps 3
+    --throughput_warmup_steps 3 \
+    --dataloader_num_workers 16
```

Comment:
We don't use the mediapipe dataloader in 1x?

Reply:
For 1x, simply increase Pytorch dataloader workers can achieve the best performance on G2. Using media dataloader are no longer improving performance.